dynamicsharingservice/policy/dynamic_policy_builder.go

```diff
@@ -0,0 +1,211 @@
+package policy
+
+import (
+	"encoding/json"
+	"strconv"
+)
+
+const (
+	attributeFamilyNameConst              = "family_name"
```

Comment:
I can see in other parts of the code we aren't using constants for attributes (like `"family_name"`) - should there be constants defined as per other SDKs to be a bit more DRY? (or is this a limitation in Go?)

Reply:
Should their scope be public or private?

Reply:
Public, see [.NET](https://github.com/getyoti/yoti-dotnet-sdk/blob/development/src/Yoti.Auth/Constants/UserProfile.cs) and [Java](https://github.com/getyoti/yoti-java-sdk/blob/cbe67f434a1326e7b65a54a49474c584a50d628c/yoti-sdk-api/src/main/java/com/yoti/api/attributes/AttributeConstants.java).

it's better for us to share them, rather than having clients having to type them in if they want to request them through Dynamic Scenario or similar 👍

Reply:
I've unified the two attribute name constant lists into one place - it was keeping them private that prevented me doing so earlier.

I can't grep any raw occurences of family_name that aren't going through the constants?

Reply:
Ok, that's fine then! 👍